src/languages/en.js

```diff
@@ -268,6 +268,7 @@ export default {
     },
     passwordForm: {
         pleaseFillOutAllFields: 'Please fill out all fields',
+        enterATwoFactorAuthenticationCodeToContinue: 'Enter a two factor authentication code to continue',
```

Comment:
NAB: I feel like `your` reads better than `a` here

Reply:
OK, I can change this.